### ISSUE TITLE
Error in get_upstream_tweets

### ISSUE THREAD

**Issue by cohenmrita** [ropensci/tidytags#80]
Hi All!

When I run get_upstream_tweets, I get the error: Error in match.names(clabs, names(xi)) : names do not match previous names.

Any help is appreciated!

**Comment by jrosen48** [comment]
Thanks @cohenmrita, two questions/requests:

1. Can you share a screen shot of the data frame you are using with this function?
2. Can you describe which steps you have taken to this point (e.g., which functions you have used before this one to generate the data frame you are using)?

**Comment by bretsw** [comment]
One additional question, just to confirm: **What version of tidytags are you using?** I ask becasue the error you're reporting reminds me a lot of the errors I was seeing when I was updating the tidytags code to v1.0 following **rtweet** updating to v1.0. The rtweet updates broke a lot of the code in v0.x versions of tidytags.

**Comment by bretsw** [comment]
Hi @cohenmrita, I was able to isolate the issue. For some reason, `rtweet::lookup_tweets()` returns different column names sometimes. I don't know why, but I've identified at least 7 different configurations of variable names in `rtweet::lookup_tweets()` output. 

Because of the different column names being returned by the same function, all kinds of issues arise when trying to combine those returned dataframes together. Basically, our approach of using `rbind()` won't work. I could update the tidytags code to use a `dplyr::join()` function, but I'd like to wait to hear back from the rtweet devs first. I've opened an issue in the rtweet repo: https://github.com/ropensci/rtweet/issues/740
